app/models/agents/digest_agent.rb

```diff
@@ -0,0 +1,55 @@
+module Agents
+  class DigestAgent < Agent
+    include FormConfigurable
+
+    default_schedule "6am"
+
+    description <<-MD
+      The Digest Agent collects any Events sent to it and emits them as a single event.
+
+      The resulting Event will have a payload message of `message`. You can use liquid templating in the `message`, have a look at the [Wiki](https://github.com/cantino/huginn/wiki/Formatting-Events-using-Liquid) for details.
+
+      Set `expected_receive_period_in_days` to the maximum amount of time that you'd expect to pass between Events being received by this Agent.
+    MD
+
+    event_description <<-MD
+      Events look like this:
+
+          {
+            "events": [ event list ],
+            "message": "Your message"
+          }
+    MD
+
+    def default_options
+      {
+          "expected_receive_period_in_days" => "2",
+          "message" => "{{ events | join: ',' }}"
```

Comment:
We don't know what payload are in `events`. So I think this is subjected to user's configuration.